advisor/backend.go

```diff
@@ -96,8 +97,8 @@ func Create() (CommandDB, error) {
 	return t, nil
 }
 
-func (t *writer) AddSnap(snapName, summary string, commands []string) error {
-	bname := []byte(snapName)
+func (t *writer) AddSnap(snapName, version, summary string, commands []string) error {
+	bname := []byte(fmt.Sprintf("%s/%s", snapName, version))
```

Comment:
I'd probably use something non-printable, eg; `\0` but this will do too.

Reply:
Yeah, we could do this as well. Should be fine though as we do not allow `/` in snapNames.

Reply:
Next time, what do we do?  name/version$revision@summary#channel? :-)

It seems cleaner if the name alone remains as a key to the bucket that contains information about the snap, and can be expanded in a structural way, probably with json if there's no performance or space pressure otherwise.